c_src/membrane_opus_plugin/_generated/nif/decoder.cpp

```diff
@@ -0,0 +1,121 @@
+#include "decoder.h"
```

Comment:
should this all bit in `.gitignore`, or should it not have been created at all?

Reply:
I am not sure but it seems that you have tried to use newer version of Unifex which generates `.cpp` files too. We can remove these files at this moment. If we switch to newer Unifex `.cpp` will be included in `.gitignore` but I think that this should be done in separate PR :slightly_smiling_face: